experimental/dds/tree2/src/test/feature-libraries/chunked-forest/chunkedForest.spec.ts

```diff
@@ -86,6 +100,38 @@ describe("ChunkedForest", () => {
 				factory: (schema) => buildChunkedForest(chunker(schema)),
 				skipCursorErrorCheck: true,
 			});
+			describe("moveCursorToPath", () => {
+				it("moves cursor to specified path.", () => {
+					const forest = buildChunkedForest(
+						chunker(new InMemoryStoredSchemaRepository(jsonDocumentSchema)),
+					);
+					initializeForest(forest, [singleJsonCursor([1, 2])]);
+
+					const cursor = forest.allocateCursor();
+					const path: UpPath = {
+						parent: undefined,
+						parentField: rootFieldKey,
+						parentIndex: 0,
+					};
+
+					forest.moveCursorToPath(path, cursor);
+					assert.deepEqual(path, cursor.getPath());
```

Comment:
we have a helper, expectEqualPaths, that is a bit more robust for doing this (it won't fail if the two paths use different implementations for example). Would be better to use it here, and in the other test below.

Reply:
updated to use expectEqualPaths